HISTORY.md

```diff
@@ -1,3 +1,9 @@
+Master
```

Comment:
```suggestion
unreleased
```

This is just the current pattern in these repos, we will be updating things in the coming year so all this is generated. Sorry about the annoyance.